bundles/AdminBundle/Controller/Admin/Asset/AssetController.php

```diff
@@ -2215,18 +2216,24 @@ public function importZipAction(Request $request)
                 ]];
             }
             $zip->close();
-        }
 
-        // here we have to use this method and not the JSON action helper ($this->_helper->json()) because this will add
-        // Content-Type: application/json which fires a download window in most browsers, because this is a normal POST
-        // request and not XHR where the content-type doesn't matter
-        $responseJson = $this->encodeJson([
-            'success' => true,
-            'jobs' => $jobs,
-            'jobId' => $jobId,
-        ]);
+            // here we have to use this method and not the JSON action helper ($this->_helper->json()) because this will add
+            // Content-Type: application/json which fires a download window in most browsers, because this is a normal POST
+            // request and not XHR where the content-type doesn't matter
+            $responseJson = $this->encodeJson([
+                'success' => true,
+                'jobs' => $jobs,
+                'jobId' => $jobId,
+            ]);
 
-        return new Response($responseJson);
+            return new Response($responseJson);
+        }
+        else {
+            return $this->adminJson([
+                'success' => false,
+                'message' => $this->trans('could_not_open_zip_file', [], 'admin')
+            ]);
```

Comment:
'admin' domain needs to be removed, as it changed from `en.extended.json` to `en.json`.
Other than that, everything works fine, ready to merge 😃!

Reply:
Does the message gets displayed properly on your side? Because it shouldn´t make a difference, since 'admin' is the default value for the domain argument. But I agree, it would be more "clean".

Reply:
ahh, it works, like you mentioned it is the default domain anyway, i overlooked that. 
Thank you for cleaning then 👍